agent/xds/testdata/listeners/connect-proxy-with-peered-upstreams.latest.golden

```diff
@@ -17,7 +17,7 @@
               "name": "envoy.filters.network.tcp_proxy",
               "typedConfig": {
                 "@type": "type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy",
-                "statPrefix": "upstream.payments?peer=cloud",
+                "statPrefix": "upstream_peered.payments.default.default.cloud",
```

Comment:
the `default.default` is aligned with existing tproxy golden files; they appear in OSS as well